### DISCUSSION TITLE
An issue with the latest version

### DISCUSSION THREAD

**Discussion by Gaerthal-Estelmist** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#38] [Q&A] [answered]
Greetings, I just signed up to say thank you for this project and to report an issue I've encountered with the latest version (1.23).
If I install the patch with the Ddraw Compat unchecked, everything seems to work fine as the previous version. I've not encountered any problems yet.
If I install the patch with the Ddraw Compat checked, I've seen significant performance improvement in the game, which is a sight for sore eyes. But now I've noticed a little glitch in the game.
Let's say there is an easel and I hover my mouse over it. Normally, without Ddraw Compat option, the in-game cursor's image would remain the same (the cursor with a * at its point) if I move it over the surface of that easel. As long as the item is highlighted, the cursor would have the same shape. But with Ddraw Compat option enabled, as I move the cursor over the highlighted item's area, it flickers and continuously changes images between the cursor with * and cursor with footsteps (the type you get when you hover the cursor over walkable area). I can still click the items without any problems but this flickering is becoming pretty annoying after a while :)
Can you please have a look and let me know if it is a problem on my side? Thanks again for this great tool!

Btw: I use 1280x720 resolution
more info: win 8.1, .net framework latest version (4.8)

EDIT2: OK, so I reinstalled the patch with the same config above and tried again. Played for a while but I could not recreate the glitch no matter what I did. I don't know what I did before to trigger that. I tried everything that came to my mind but it just has not happened again. I'll keep an eye out for it and write the details here if I find more about it. Playing the game with high FPS is so satisfying!

**Comment by FaithBeam** [comment, marked as answer]
@Gaerthal-Estelmist That's pretty wacky! I was not able to recreate that specific glitch but I do understand ddrawcompat may have some weird visual bugs. I can confirm screen smearing with ddrawcompat while playing at 4k: https://www.youtube.com/watch?v=qOVu_brmK0o

I don't expect anyone to actually play at that high of resolution, but I'd like to give an option for wrappers in case they experience visual glitches such as yours. I plan to add a drop down that you can select ddrawcompat or dgvoodoo2.

**Reply by Gaerthal-Estelmist** [reply]
Thank you for the prompt reply. The one I reported is nowhere near trippy or serious an issue like the one in your video,  I think I should be thankful I just got a minor cursor issue :)
Another thing I just discovered, up until version 1.23, it would break the in-game dialogue boxes and the images if I alt-tabbed out of the game and went back in. For this reason, I could not use alt-tab even though it is very important for me. Now, as I tried alt-tabbing with the ddraw compat on, it no longer breaks the in-game images! Everything works perfectly after getting back to the game! I think this is related to the ddraw compat because I never used that option before. dgvoodoo2 may have worked, but I did not know what it was so I never gave it a try. I'm glad I tried ddraw option. Finally after all these years I can play this game of my childhood with all its glory! Can't thank you enough!

I'll keep an eye out for any glitches like the one I reported before and let you know here. Keep up the good work, and have a good weekend!

**Comment by libertyfreeform** [comment]
Hi there, thank you for your work! Intalled the patcher [Sims1WidescreenPatcher.3.13.0+win.x64] to take high rez screenshots (in my case 2560x1440 16:9 ddraw compat). Win+Alt+Prtsc always worked before now it captures just black screen. How can I fix it?
